src/main/java/net/sf/jabref/logic/autocompleter/NameFieldAutoCompleter.java

```diff
@@ -93,7 +94,7 @@ public void addBibtexEntry(BibEntry entry) {
             if (entry.hasField(fieldName)) {
                 String fieldValue = entry.getField(fieldName);
                 AuthorList authorList = AuthorList.getAuthorList(fieldValue);
-                for (AuthorList.Author author : authorList.getAuthorList()) {
+                for (Author author : authorList.getAuthorList()) {
```

Comment:
AuthorList.getAuthorList() sounds really redundant from naming.
Together with the next line it is confusing on the first read.
What about a simple  `getAuthors()`